services/src/unit-test/java/uk/gov/moj/sdt/services/cache/ErrorMessagesCacheTest.java

```diff
@@ -49,123 +52,120 @@
  *
  * @author d130680
  */
-public class ErrorMessagesCacheTest extends AbstractSdtUnitTestBase {
-
-    /**
-     * Error messages cache.
-     */
-    private ErrorMessagesCache cache;
+@ExtendWith(MockitoExtension.class)
+class ErrorMessagesCacheTest extends AbstractSdtUnitTestBase {
 
     /**
-     * Management bean holding cache information.
+     * Generic dao.
      */
-    private ISdtManagementMBean managementMBean;
+    @Mock
+    private IGenericDao mockGenericDao;
 
     /**
-     * Generic dao.
+     * Error messages cache.
      */
-    private IGenericDao mockGenericDao;
+    private ErrorMessagesCache cache;
 
     /**
      * Dao query results.
      */
     private ErrorMessage[] result;
 
+    private static final String ERROR_DESCRIPTION_1 = "errorDescription 1";
+    private static final String ERROR_DESCRIPTION_2 = "errorDescription 2";
+    private static final String ERROR_DESCRIPTION_3 = "errorDescription 3";
+
     /**
      * Setup mock objects and inject DAO dependencies into our class.
      */
-    @Before
+    @BeforeEach
+    @Override
     public void setUp() {
+         MockitoAnnotations.openMocks(this);

```

Comment:
This should be able to be removed as the ExtendWith(MockitoExtension.class) annotation initialises the annotated mocks in the class.

Reply:
removed

Reply:
Confirmed that openMocks() line has been removed.